src/components/token/Tokens.js

```diff
@@ -97,6 +116,7 @@ class Tokens extends React.Component {
                 }
             ]
         });
+        this.updateURL();
```

Comment:
I would just add a comment here that this will update the URL when the search text, sort, or sort order changes. I had to look at the `tableHeaderClicked` method to understand it also calls `onSearchButtonClicked`.